docs/iot-video-tutorials.md

```diff
@@ -57,3 +57,57 @@ You will learn how to:
         <iframe src="https://www.youtube.com/embed/JCW_hShAp7I" frameborder="0" allowfullscreen></iframe>
     </div>
 </div>
+
+
+#### Dashboard development guide, Part 1: Visualizing Assets data using Maps and Tables
+
+The first part of this tutorial provides preparations for the creating a new dashboard, visualizing attributes data.
+
+You will learn how to:
+
+- create assets and devices, build relations between them. 
+- add server attributes and create a new dashboard.
+- visualize attributes data using Entities Table and Map widgets.
+
+<div id="video">  
+    <div id="video_wrapper">
+        <iframe src="https://www.youtube.com/embed/BhYnKtbUHHI" frameborder="0" allowfullscreen></iframe>
+    </div>
+</div>
+
+
+#### Dashboard development guide, Part 2: Dashboard states, Aliases and Widget actions
+
+The second part of this tutorial covers basic operations with dashboard states and actions, introduces us to aliases and some other widgets.
```

Comment:
The second part of this tutorial covers basic operations with dashboard states and actions, introduces us to entity aliases and image map widget.